### ISSUE TITLE
Woo Payments: "Activate" label on form toggles does not activate the toggle

### ISSUE THREAD

**Issue by saramarcondes** [Automattic/wp-calypso#47018]
#### Steps to reproduce

1. Starting at URL: `/store/settings/payments/:site`
2. Hover your mouse over the "activate" button for one of the payment methods (like PayPal or ACH)
3. Note that the mouse turns to a clickable pointer finger
4. Click the lable

#### What I expected
The form toggle should be toggled

#### What happened instead
Nothing happens

#### Browser / OS version
Firefox, macOS

#### Screenshot / Video
![](https://recordit.co/dmKbqsUXyo.gif)

Note: In the GIF you don't see the cursor change to a pointer but in actuality it does.

#### Context / Source

Found while testing https://github.com/Automattic/wp-calypso/pull/46945

**Comment by synora** [comment]
As per the labels, this is related to Store. We're sunsetting Store by Feb 2021. Closing it in favor of that.
